src/doc/rustc/src/targets/known-issues.md

```diff
@@ -0,0 +1,7 @@
+# Known Issues
+
+Cross compiling your code to different architectures via the target-feature is generally considered as unsafe. Below a list with known "gotchas".
+
+| Target-Feature | Issue | Seen on | Description | Details |
+| -------------- | ----- | ------- | ----------- | ------- |
+| `soft-float` | Segfaults and ABI mismatches | `x86_64-unknown-linux-gnu` | Using soft-floats disables usage of ```xmm``` registers, but parts of ```std``` are compiled without soft-floats and expect parameters to be passed in ```xmm``` registers. This leads to ABI mismatches. If you choose to use target-features on functions via `#[target_feature]`, keep in mind that the function is then unsafe. Mixing crates with different "global" target-features can lead to ABI issues, too. | [#63466](https://github.com/rust-lang/rust/issues/63466)
```

Comment:
This part:

> If you choose to use target-features on functions via `#[target_feature]`, keep in mind that the function is then unsafe. Mixing crates with different "global" target-features can lead to ABI issues, too.

Is true (and relevant) in general, not specific to this one target feature or #63466. So I don't think it belongs into this table, but perhaps into the introductory text above.

As for the other columns: There's nothing Linux-specific about this problem, and it'll also occur on (most) 32 bit x86 targets, not just on x86-64 ones. so I think the "Seen on" column should just say "x86 and x86-64".